src/components/syncPlay/groupSelectionMenu.js

```diff
@@ -37,7 +37,7 @@ function showNewJoinGroupSelection (button, user, apiClient) {
         console.debug('No item is currently playing.');
     }
 
-    apiClient.sendSyncPlayCommand(sessionId, 'ListGroups').then(function (response) {
+    apiClient.sendSyncPlayCommand('ListGroups').then(function (response) {
```

Comment:
@MrTimscampi doesn't this need an update to apiclient?..

Reply:
It's done already, the PR there (and this one) are waiting for the backend PR.

Reply:
This PR should update the version of apiclient used.

Reply:
We need to release that first, but that needs the backend PR to be merged first.

Reply:
I believe this was addressed below.